storage-mirror-firestore/stress_test/bin/driver.ts

```diff
@@ -711,6 +711,135 @@ const backfillCmd = new commander.Command("backfill")
     spinner.succeed(`Mirrored ${succeeded} paths in ${elapsed}s.`);
   });
 
+const cleanTombstonesCommand = new commander.Command("clean-tombstones")
+  .allowUnknownOption(false)
+  .description("delete tombstone records from firestore")
+  .option(
+    "--project <project_id>",
+    "project id which has the extension installed"
+  )
+  .option(
+    "--instance-id <extension_instance_id>",
+    "instance id of the extension (which can be found with `firebase ext:list`).",
+    "storage-mirror-firestore"
+  )
+  .action(async (command: commander.Command) => {
+    interface CommandWithOpts extends commander.Command {
+      project?: string;
+      instanceId?: string;
+    }
+    interface ExtensionInstance {
+      state: string;
+      config: {
+        params: {
+          ITEMS_TOMBSTONES_NAME: string;
+          PREFIXES_TOMBSTONES_NAME: string;
+          FIRESTORE_ROOT: string;
+        };
+      };
+    }
+
+    const options = command as CommandWithOpts;
+    const auth = new GoogleAuth({
+      scopes: ["https://www.googleapis.com/auth/firebase.readonly"],
+    });
+    const authClient = await auth.getClient();
+    const projectId = options.project || (await auth.getProjectId());
+
+    let instance: ExtensionInstance;
+    let url = `https://firebaseextensions.googleapis.com/v1beta/projects/${projectId}/instances/${options.instanceId}`;
+    try {
+      const response = await authClient.request({ url });
+      instance = response.data as ExtensionInstance;
+    } catch (e) {
+      if (e.code === 404) {
+        console.error(
+          `Extension instance '${options.instanceId}' on project '${projectId}' was not found.`
+        );
+        process.exit(1);
+      }
+      throw e;
+    }
+
+    if (instance.state !== "ACTIVE") {
+      console.error(
+        `Extension instance '${options.instanceId}' on project '${projectId}' is not active.`
+      );
+      return;
+    }
+
+    const prompt = `Are you sure you want to delete tombstone records from Firestore for the project '${projectId}' and instance '${options.instanceId}'?`;
+    if (!(await confirm(prompt))) {
+      process.exit(0);
+    }
+
+    console.log("\nScanning project for tombstone records...\n");
+
+    admin.initializeApp({
+      credential: admin.credential.applicationDefault(),
```

Comment:
This will require a developer to add an environment variable with credentials, correct? If so, please add to docs

Reply:
As above re-backfill and other scripts - since a few of the scripts need this I wonder if the POSTINSTALL.md needs a good restructuring either now or if we're extracting the user executable scripts out of stress_test - thoughts?

Reply:
Doing a restructure after the fact makes sense to me. I filed https://github.com/FirebaseExtended/experimental-extensions/issues/59 so we don't forget